src/pybind/mgr/cephadm/module.py

```diff
@@ -1191,6 +1201,67 @@ def run(h: str) -> str:
 
         return HandleCommandResult(stdout='\n'.join(run(host)))
 
+    @orchestrator._cli_read_command('orch client-keyring ls')
```

Comment:
are we okay with starting to put "ceph orch" commands directly in the cephadm module rather than the orchestrator? Other commands we've done here have been "ceph cephadm"